src/developer/web-api/openapi.md

```diff
@@ -42,4 +42,22 @@ Available tags are:
 * `integration`
 * `login`
 * `query`
-* `management`
\ No newline at end of file
+* `management`
+
+All endpoints that generate a OpenAPI document support the following optional 
+request parameters:
+
+### `failOnNameClash`
+When set to `true`, two or more types of same simple (unqualified) name are considered clashing and the generation fails with an error. 
+
+When set `false` (default), name clashes are resolved by adding numbers to the simple name to make each of them unique.
+As a result the names are not predictable or stable. Merging with documentation based on name will be broken. 
+This option is meant as a preview feature. 
+Request made to produce a document that is used further should be generated with the parameter set to `true`.
```

Comment:
I am not sure I understand `used further` here. If I understand correctly maybe this

```suggestion
This option is meant as a preview feature which should only be used during development.
```

is clearer 🤔